client/src/pages/QuizPage.jsx

```diff
@@ -1,10 +1,10 @@
 import React, { useState, useEffect } from "react";
 import { useParams } from "react-router-dom";
 import axios from "axios";
-import Footer from "../components/footer/footer";
-import Header from "../components/header/index";
-import Success from "../components/success";
-import Option from "../components/option-botton";
+import Footer from "../components/Footer/Footer";
```

Comment:
it should be:  `import Footer from "../components/Footer";`

porque al apuntar directamente a la carpeta... el compilador automaticamente toma el index.js que trae esa carpeta

Reply:
Update folder directory format